code/modules/virus2/effect.dm

```diff
@@ -834,305 +723,410 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
+	to_chat(mob, "<span class = 'notice'>You feel delightful!</span>")
+	if (mob.reagents.get_reagent_amount(DOCTORSDELIGHT) < 1)
+		mob.reagents.add_reagent(DOCTORSDELIGHT, 1)
 
 
-/datum/disease2/effect/drool
-	name = "Saliva Effect"
-	stage = 1
+/datum/disease2/effect/spawn
+	name = "Arachnogenesis Effect"
+	stage = 4
+	var/spawn_type=/mob/living/simple_animal/hostile/giant_spider/spiderling
+	var/spawn_name="spiderling"
 
-/datum/disease2/effect/drool/activate(var/mob/living/carbon/mob)
-	mob.say("*drool")
+/datum/disease2/effect/spawn/activate(var/mob/living/carbon/mob)
+	//var/mob/living/carbon/human/H = mob
+	var/placemob = locate(mob.x + pick(1,-1), mob.y, mob.z)
+	playsound(mob.loc, 'sound/effects/splat.ogg', 50, 1)
 
+	new spawn_type(placemob)
+	mob.emote("me",1,"vomits up a live [spawn_name]!")
 
-/datum/disease2/effect/twitch
-	name = "Twitcher"
-	stage = 1
+/datum/disease2/effect/spawn/roach
+	name = "Blattogenesis Effect"
+	stage = 4
+	spawn_type=/mob/living/simple_animal/cockroach
+	spawn_name="cockroach"
 
-/datum/disease2/effect/twitch/activate(var/mob/living/carbon/mob)
-	mob.say("*twitch")
 
+/datum/disease2/effect/orbweapon
+	name = "Biolobulin Effect"
+	stage = 4
+/datum/disease2/effect/orbweapon/activate(var/mob/living/carbon/mob)
+	var/obj/item/toy/snappop/virus/virus = new /obj/item/toy/snappop/virus
+	mob.put_in_hands(virus)
 
-/datum/disease2/effect/headache
-	name = "Headache"
-	stage = 1
 
-/datum/disease2/effect/headache/activate(var/mob/living/carbon/mob)
-	to_chat(mob, "<span class = 'notice'>Your head hurts a bit</span>")
+/datum/disease2/effect/plasma
+	name = "Toxin Sublimation"
+	stage = 4
 
+/datum/disease2/effect/plasma/activate(var/mob/living/carbon/mob)
+	//var/src = mob
+	var/hack = mob.loc
+	var/turf/simulated/T = get_turf(hack)
+	if(!T)
+		return
+	var/datum/gas_mixture/GM = new
+	if(prob(10))
+		GM.toxins += 100
+		//GM.temperature = 1500+T0C //should be enough to start a fire
+		to_chat(mob, "<span class='warning'>You exhale a large plume of toxic gas!</span>")
+	else
+		GM.toxins += 10
+		GM.temperature = istype(T) ? T.air.temperature : T20C
+		to_chat(mob, "<span class = 'warning'> A toxic gas emanates from your pores!</span>")
+	T.assume_air(GM)
+	return
 
-/datum/disease2/effect/itching
-	name = "Itching"
-	stage = 1
 
-/datum/disease2/effect/itching/activate(var/mob/living/carbon/mob)
-	var/mob/living/carbon/human/H = mob
-	if (istype(H) && H.species && H.species.flags & NO_SKIN)
-		to_chat(mob, "<span class='warning'>Your bones itch!</span>")
-	else
-		to_chat(mob, "<span class='warning'>Your skin itches!</span>")
+/datum/disease2/effect/babel
+	name = "Babel Syndrome"
+	stage = 4
+	max_count = 1
 
+	var/list/original_languages = list()
 
-/datum/disease2/effect/drained
-	name = "Drained Feeling"
-	stage = 1
+/datum/disease2/effect/babel/activate(var/mob/living/carbon/mob)
+	if(mob.languages.len <= 1)
+		to_chat(mob, "Your knowledge of language is just fine.")
```

Comment:
Ehhh
It makes more sense out of context, sure
Might want to find a new message instead